project/src/main/webapp/voting_edu.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en-US">
+  <head>
+    <meta charset="utf-8">
+    <meta name="Voter Central Homepage"
+          content="The homepage of Voter Central, a website
+                   for centralizing voting information">
+    <link rel="stylesheet" href="style.css">
+    <script src="config.js"></script>
+    <script src="script.js"></script>
+    <title>Voter Central Homepage</title>
+  </head>
+  <body>
+    <h2>How to vote</h2>
+    <iframe width="560" height="315" src="https://www.youtube.com/embed/R2yFIPRZXu4" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Break this long line into multiple lines. Indent +4 spaces at all wrapped lines.
The same applies for the two iframes below as well.